examples/xgboost/iris-classifier/predictor.py

```diff
@@ -5,17 +5,20 @@
 labels = ["iris-setosa", "iris-versicolor", "iris-virginica"]
```

Comment:
Should this be in `init()`? I'm not sure that it should, just wanted you to consider